lib/community/digest2/personalizeData.js

```diff
@@ -82,8 +82,9 @@ const personalizeData = (user, data, opts = {}) => {
     },
     email_settings_url: Frontend.Route.userSettings() + clickthroughParams + '&expand=account',
     tracking_pixel_url: Analytics.pixelUrl('Digest', {userId: user.id, community: data.community_name}),
-    form_action_url: Frontend.Route.emailPostForm(),
```

Comment:
I've created an Asana subtask to rename. I was thinking we were switching not AB testing, so wasn't an issue.